lambdas/lambdas.mk

```
@@ -38,6 +39,10 @@ config: .chalice/config.json
 .PHONY: environ
 environ: vendor/resources/environ.json
 
+.PHONY: swagger
+swagger:
+	python $(root_dir)/scripts/make_swagger_ui.py
```

```suggestion
	python $(project_root)/scripts/make_swagger_ui.py
```

which check_python ensures is set